form-samples/two-column-logo-header/README.md

```diff
@@ -0,0 +1,39 @@
+# Dashed Line Header
+
+## Summary
+This sample displays the `Title` and `Logo` values in the heading and an additional 'Description' below the 'Title'. If the `Description` value is empty, it will be hidden.
```

Comment:
The `Logo` and `Description` columns were not included in the JSON. Please review the description or JSON.